clustershell.spec.in

```diff
@@ -154,15 +161,16 @@ rm -rf %{buildroot}
 %{_bindir}/nodeset
 %{python2_sitelib}/ClusterShell/
 %{python2_sitelib}/ClusterShell-*-py?.?.egg-info
+%endif
 
 %files -n %{python3_pkgprefix}-%{name}
 %if 0%{?rhel}
 %defattr(-,root,root,-)
 %endif
-%{_bindir}/clubak-%{python3_shortver}
-%{_bindir}/cluset-%{python3_shortver}
-%{_bindir}/clush-%{python3_shortver}
-%{_bindir}/nodeset-%{python3_shortver}
+%{_bindir}/clubak
+%{_bindir}/cluset
+%{_bindir}/clush
+%{_bindir}/nodeset
```

Comment:
Thanks for these changes. We're almost there. You will still need to keep a if condition here to install the versioned version when not installing on rhel8.